### DISCUSSION TITLE
how do i remove all the weeks?

### DISCUSSION THREAD

**Discussion by imdumlol** [CodenameCrew/CodenameEngine#47] [Q&A]
idk how

**Comment by biomseed** [comment]
if there's no weeks then what will happen if you enter story mode?

**Comment by imdumlol** [comment]
crash

**Comment by imdumlol** [comment]
but i want to remove them (for a port)

**Reply by biomseed** [reply]
> but i want to remove them (for a port)

is this a source mod or a softcoded mod?

**Comment by NexIsDumb** [comment]
cant you just add a mod? like whenever you add a mod the og weeks disappear
(im very late😭)

**Comment by THEkatinamicrowave** [comment]
Why is this still at the top of the discussions page

**Reply by NexIsDumb** [reply]
idfk actually

**Reply by biomseed** [reply]
It depends on what sorting option is picked

**Reply by r6915ee** [reply]
I have no clue.
